Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -27,53 +27,83 @@ static NSString *const STPSDKVersion = @"11.0.0";
 @interface Stripe : NSObject FAUXPAS_IGNORED_ON_LINE(UnprefixedClass);
 
 /**
- *  Set your Stripe API key with this method. New instances of STPAPIClient will be initialized with this value. You should call this method as early as
- *  possible in your application's lifecycle, preferably in your AppDelegate.
- *
- *  @param   publishableKey Your publishable key, obtained from https://stripe.com/account/apikeys
- *  @warning Make sure not to ship your test API keys to the App Store! This will log a warning if you use your test key in a release build.
+ Set your Stripe API key with this method. New instances of STPAPIClient will be initialized with this value. You should call this method as early as
+ possible in your application's lifecycle, preferably in your AppDelegate.
+
+ @param   publishableKey Your publishable key, obtained from https://stripe.com/account/apikeys
+ @warning Make sure not to ship your test API keys to the App Store! This will log a warning if you use your test key in a release build.
  */
 + (void)setDefaultPublishableKey:(NSString *)publishableKey;
 
-/// The current default publishable key.
+/**
+ The current default publishable key.
+ */
 + (nullable NSString *)defaultPublishableKey;
 
 @end
 
-/// A client for making connections to the Stripe API.
+/**
+ A client for making connections to the Stripe API.
+ */
 @interface STPAPIClient : NSObject
 
 /**
- *  A shared singleton API client. Its API key will be initially equal to [Stripe defaultPublishableKey].
+ A shared singleton API client. Its API key will be initially equal to [Stripe defaultPublishableKey].
  */
 + (instancetype)sharedClient;
+
+
+/**
+ Initializes an API client with the given configuration. Its API key will be 
+ set to the configuration's publishable key.
+
+ @param configuration The configuration to use.
+ @return An instance of STPAPIClient.
+ */
 - (instancetype)initWithConfiguration:(STPPaymentConfiguration *)configuration NS_DESIGNATED_INITIALIZER;
+
+/**
+ Initializes an API client with the given publishable key.
+
+ @param publishableKey The publishable key to use.
+ @return An instance of STPAPIClient.
+ */
 - (instancetype)initWithPublishableKey:(NSString *)publishableKey;
 
 /**
- *  @see [Stripe setDefaultPublishableKey:]
+ The client's publishable key.
  */
 @property (nonatomic, copy, nullable) NSString *publishableKey;
 
 /**
- *  @see -initWithConfiguration
+ The client's configuration.
  */
 @property (nonatomic, copy) STPPaymentConfiguration *configuration;
 
+
+/**
+ In order to perform API requests on behalf of a connected account, e.g. to
+ create a source on a connected account, set this property to the ID of the 
+ account for which this request is being made.
+
+ @see https://stripe.com/docs/sources/connect#creating-direct-charges
```

Comment:
Ah, good call. I'll point to the direct charges link in `createSource` and mention setting `stripeAccount`.